examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/form/fragment/_cq_template.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:fd="http://www.adobe.com/aemfd/fd/1.0"
     jcr:primaryType="nt:unstructured"
     jcr:title="Fragment"
+    fd:viewType="fragment"
```

Comment:
This would change the :type value in the exported JSON, are we consuming this in headless ?

Reply:
Also, since fd:viewType would always be fragment, can we rather add this in the java code also, as a fallback we can consume fd:viewType if required

Reply:
I don't think we are using `:type` for fragment in headless. 

Okay, I'm adding in model JSON. I can't see a way to hardcode this property in template,  please help with that.
As the objective of adding fd:viewType in `cq:template` is to identify if the give component is "Fragment" , but not sure where I should hard-code the same.